lib/tracker_api/resources/story.rb

```diff
@@ -121,6 +122,18 @@ def owners(params = {})
         end
       end
 
+      # Provides a list of all the transitions of the story.
+      #
+      # @param [Hash] params
+      # @return [Array[StoryTransition]]
+      def story_transitions(params = {})
```

Comment:
This should be named `transitions` so that it is an override of the getter. The getter is implementing a lazy load of the transitions.